openapi.yaml

```diff
@@ -606,6 +665,26 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/VehicleCategory'
+    Equipment:
+      type: object
+      properties:
+        name:
+          type: string
+          description: Name of the equipment item or package.
+        packageItems:
+          type: array
+          description: Equipment items included in the package.
+          items:
+            $ref: '#/components/schemas/PackageItem'
+          nullable: true
+    PackageItem:
+      type: object
+      properties:
+        name:
+          type: string
+          description: Name of the equipment item.
+          example: Adaptive Cruise Control
+
```

Comment:
when we are happy with this PR, i will follow it up with alphabetical ordering of all the schemas too